src/com/itacademy/eshop/simulations/ManagerSimulation.java

```diff
@@ -28,24 +28,33 @@ private void addThreeNewProducts() {
          * adds three new products to the shop. One of them should be a book, one should be a Laptop, and one should be a shirt.
          */
         Product laptop = new Product("Laptop", 1000, Category.ELECTRONICS);
+        Product shirt = new Product("Shirt", 400, Category.CLOTHING);
+        Product book = new Product("Book", 300, Category.BOOKS);
+
+        shop.addProduct(laptop);
+        shop.addProduct(shirt);
+        shop.addProduct(book);
     }
 
     private void findAndRemoveOneProduct() {
         /**
          * manager searcher for a product with the name "Laptop" (the one added a moment before) and removes it from the shop
          */
+        shop.removeProductByName("Laptop");
     }
 
     private void changePriceForOneProduct() {
         /**
          * manager searches for a product with the name "Shirt" (the one added a moment before) and changes its price to 20
          */
+        shop.changePriceForProduct("Shirt",55);
     }
 
     private void removeAllProductsWithCategory(Category category) {
         /**
          * manager removes all products with the category FOOD
          */
+        shop.removeProductByCategory(Category.FOOD);
```

Comment:
instead of Category.FOOD use just category parameter which at current moment serves no purpose